src/rest.cpp

```diff
@@ -617,7 +619,7 @@ static bool rest_deploymentinfo(const std::any& context, HTTPRequest* req, const
 
         if (!hash_str.empty()) {
             uint256 hash;
-            if (!ParseHashStr(hash_str, hash)) {
+            if (!hash.SetHex(hash_str)) {
```

Comment:
Noticed `ParseHashV(hash_str, ...)` is needlessly called a few lines below instead of using the already verified `hash` value.